### PR TITLE
update: text edits from Swedish to English

### PR DESCRIPTION
**Pull request by EugeneJoe**

### CODE REVIEW

File: sb.web.app/src/components/CreateArticle/NewArticle.vue [hunk 1]
```diff
@@ -280,7 +280,7 @@ export default {
   
 .center-text {
   text-align: center;
-  margin-top: 4rem;
+  magin-top: 4rem;
```
Comment (utunga) [inline sb.web.app/src/components/CreateArticle/NewArticle.vue:283]: This is a misspelling? Huh?

File: sb.web.app/src/components/CreateArticle/StepTwo.vue [hunk 2]
```diff
@@ -80,6 +80,28 @@ export default {
...
+      let placeholder = ''
+      if (this.savedProgress.status === 'Offer') {
+        placeholder = ' of this ' + this.chosenType.toLowerCase() + '?'
+      } else {
+        placeholder = ' you are looking to pay?'
+      }
+
+      return placeholder
+    }
+  },
```
Comment (utunga) [inline sb.web.app/src/components/CreateArticle/StepTwo.vue:104]: fwiw the i18n library has special functions for doing this kind of thing... that would let us keep all that logic in the translation side of things..

File: sb.web.app/src/components/CreateArticle/NewArticle.vue (hunk 1 above)
Comment (utunga) [reply sb.web.app/src/components/CreateArticle/NewArticle.vue:283]: I guess just a typo that slipped through.

File: sb.web.app/src/components/CreateArticle/StepTwo.vue (hunk 2 above)
Comment (utunga) [reply sb.web.app/src/components/CreateArticle/StepTwo.vue:104]: Take a look at some of the more complex examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that is not English specific 

Also if you want to access the $i18n t method from code (as in not in the vue template) you can do it this way 
`this.$i18n.t('time.now')`

File: sb.web.app/src/components/Navbar/Notifications.vue [hunk 3]
```diff
@@ -33,7 +33,7 @@
                     <router-link :to="{name:'Profile', params:{tab: 'purchases'}}" @click.prevent="">
                         <div id="new-list-content">
                         <img class="notice-img" src="../../assets/navbar_logos/notice.png" alt="{{ $t('notifications.new_notification') }}"/>
-                        <p class="notice-desc">{{ item.fromUser }} har godkänt din köpförfrågan{{ $t('notifications.go_to') }} <u> {{ $t('nav.my_account') }} </u> {{ $t('notifications.download_invoice') }}</p>
+                        <p class="notice-desc">{{ item.fromUser }} has approved your purchase request{{ $t('notifications.go_to') }} <u> {{ $t('nav.my_account') }} </u> {{ $t('notifications.download_invoice') }}</p>
```
Comment (utunga) [inline sb.web.app/src/components/Navbar/Notifications.vue:36]: Sorry but this needs to be done via i18n - we need to be able to find *all* the English that needs to be transalted back to Swedish later .. if we do it like this we're gonna have the reverse problem of having to find allt he English to remove later on. 


Take a look at some of the mor examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that is not English specific

File: sb.web.app/src/components/Navbar/Notifications.vue [hunk 4]
```diff
@@ -51,7 +51,7 @@
                     <router-link :to="{name:'Chat', params:{chatID: item.chatID}}" @click.prevent="">
                         <div id="new-list-content">
                         <img class="notice-img" src="../../assets/navbar_logos/notice.png" alt="{{ $t('notifications.new_notification') }}"/>
-                        <p class="notice-desc">{{ item.fromUser }} {{ $t('notifications.have_sent') }}ett chatt-meddelande.</p>
+                        <p class="notice-desc">{{ item.fromUser }} {{ $t('notifications.have_sent') }}a chat message.</p>
```
Comment (utunga) [inline sb.web.app/src/components/Navbar/Notifications.vue:54]: Sorry but this needs to be done via i18n - we need to be able to find *all* the English that needs to be transalted back to Swedish later .. if we do it like this we're gonna have the reverse problem of having to find allt he English to remove later on. 


Take a look at some of the mor examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that is not English specific

File: sb.web.app/src/components/Navbar/Notifications.vue [hunk 5]
```diff
@@ -80,7 +80,7 @@
                     <div v-if="item.type == 'saleRequestAccepted'">
                     <router-link :to="{name:'Profile', params:{tab: 'purchases'}}" >
                         <div id="new-list-content">
-                        <p class="notice-desc">{{ item.fromUser }} har godkännt din köpförfrågan{{ $t('notifications.go_to') }} <u> {{ $t('nav.my_account') }} </u> {{ $t('notifications.download_invoice') }}</p>
+                        <p class="notice-desc">{{ item.fromUser }} has approved your purchase request{{ $t('notifications.go_to') }} <u> {{ $t('nav.my_account') }} </u> {{ $t('notifications.download_invoice') }}</p>
```
Comment (utunga) [inline sb.web.app/src/components/Navbar/Notifications.vue:83]: Sorry but this needs to be done via i18n - we need to be able to find *all* the English that needs to be transalted back to Swedish later .. if we do it like this we're gonna have the reverse problem of having to find allt he English to remove later on. 


Take a look at some of the mor examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that is not English specific

File: sb.web.app/src/components/Navbar/Notifications.vue [hunk 6]
```diff
@@ -96,7 +96,7 @@
                     <div v-if="item.type == 'chatMessage'">
                     <router-link :to="{name:'Chat', params:{chatID: item.chatID}}" >
                         <div id="new-list-content">
-                        <p class="notice-desc">{{ item.fromUser }} {{ $t('notifications.have_sent') }}ett chatt-meddelande.</p>
+                        <p class="notice-desc">{{ item.fromUser }} {{ $t('notifications.have_sent') }}a chat message.</p>
```
Comment (utunga) [inline sb.web.app/src/components/Navbar/Notifications.vue:99]: Sorry but this needs to be done via i18n - we need to be able to find *all* the English that needs to be transalted back to Swedish later .. if we do it like this we're gonna have the reverse problem of having to find allt he English to remove later on. 


Take a look at some of the mor examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that is not English specific

File: sb.web.app/src/components/Profile/MyProfile.vue [hunk 7]
```diff
@@ -47,7 +47,7 @@
       <form className="flexbox-container2" @submit.prevent="">
         <div className="container-item">
           <h1>{{ $t('user.general_information') }}</h1>
-          <label for="logo">Logotyp:</label><br/>
+          <label for="logo">Profile Picture:</label><br/>
```
Comment (utunga) [inline sb.web.app/src/components/Profile/MyProfile.vue:50]: Sorry but this needs to be done via i18n not hardcoded to English

File: sb.web.app/src/locales/en.json [hunk 8]
```diff
@@ -153,7 +153,7 @@
         "purchase_request": "You have received a purchase request from",
         "go_to": ". Go to",
         "approve_or_decline": "to approve or decline.",
-        "have_sent": "have sent",
+        "have_sent": "has sent",
```
Comment (utunga) [inline sb.web.app/src/locales/en.json:156]: Take a look at some of the more complex examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that can avoid these weird 'sentence fragments' lying around

**Review by utunga — CHANGES_REQUESTED** [review]
Marking this as request changes - can't merge as is sorry

File: sb.web.app/src/components/CreateArticle/NewArticle.vue (hunk 1 above)
Comment (utunga) [reply sb.web.app/src/components/CreateArticle/NewArticle.vue:283]: If you can fix it back to how it was then come back

File: sb.web.app/src/locales/en.json (hunk 8 above)
Comment (EugeneJoe) [reply sb.web.app/src/locales/en.json:156]: Okay.
I figured we were doing a full switch to English.
I'll chuck them into the en.josn and use that for all these scenarios

Comment (EugeneJoe) [reply sb.web.app/src/locales/en.json:156]: We'll kind of have to hard-code some of these values in the code if we end up using this named formatting method.

File: sb.web.app/src/components/CreateArticle/StepTwo.vue (hunk 2 above)
Comment (EugeneJoe) [reply sb.web.app/src/components/CreateArticle/StepTwo.vue:104]: Won't we end up chucking translations into the html code if I use named formatting? @utunga

File: sb.web.app/src/locales/se.json [hunk 9]
```diff
@@ -119,6 +120,10 @@
         "the_seller": "Säljaren",
         "whatDoYouWantToSearchForTodayLabel": "Vad vill du söka efter idag?"
     },
+    "notifications": {
+        "approved_purchase": "har godkänt din köpförfrägan",
```
Comment (utunga) [inline sb.web.app/src/locales/se.json:124]: are these keys also in en.json ?

**Review by utunga — CHANGES_REQUESTED** [review]
Still a couple more changes wanted.. i sent as telegram comment

File: sb.web.app/src/locales/se.json (hunk 9 above)
Comment (EugeneJoe) [reply sb.web.app/src/locales/se.json:124]: yes.
Added them in both

**Review by utunga — APPROVED** [review]
Approving so we can get these changes deployed. It's better like this than not. 

But we don't want *any* English *or* Swedish text in the code if we can avoid it.

**Pull request merged**